maths/sigmoid.py

```diff
@@ -36,5 +36,4 @@ def sigmoid(vector: np.array) -> np.array:
 
 if __name__ == "__main__":
     import doctest
-    
```

Comment:
Please run psf/black on this code as discussed in CONTRIBUTING.md because it will reverse this change.

Reply:
@cclauss Thank you. I modified the code.